app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='login-div'>
```

Comment:
### Optional
- [ ] It would be nice if we can log out of the application like all other apps :man_technologist: